evutil.c

```diff
@@ -2162,6 +2170,42 @@ evutil_inet_ntop(int af, const void *src, char *dst, size_t len)
 #endif
 }
 
+static int
+evutil_inet_pton_scope(int af, const char *src, void *dst, unsigned *indexp)
+{
+	int r;
+	unsigned if_index;
+	char *check, *cp, *tmp_src;
+
+	*indexp= 0;	/* Reasonable default */
+
+	/* Bail out if not IPv6 */
+	if (af != AF_INET6)
+		return evutil_inet_pton(af, src, dst);
+
+	cp= strchr(src, '%');
```

Comment:
Missing spaces on both sides of the equal sign.
You can check the entire function using `clang-format -i --lines=2173:2208 evutil.c`.